aws/notebook/.daskernetes.yaml

```diff
@@ -0,0 +1,27 @@
+worker:
+  spec:
+    containers:
+    - args: [dask-worker, --nthreads, '2', --no-bokeh, --memory-limit, 8GB, --death-timeout, '60']
+      image: daskdev/pangeo-worker:latest
+      name: dask-worker
+      security_context:
+        capabilities:
+          add: [SYS_ADMIN]
+        privileged: true
+      volume_mounts:
+        - mountPath: /dev/fuse
+          name: fuse
+      env:
+        - name: EXTRA_PIP_PACKAGES
+          value: git+https://github.com/dask/distributed
+        - name: GCSFUSE_BUCKET
```

Comment:
Does this translate to S3 bucket?  Use a more generic name for this environment variable?

Reply:
AWS can read from GCS, albeit less performantly, and I don't suppose there is a desire to duplicate the data. Having said that, there will be a need to transfer some form of credentials, if not on google hardware. 

There are s3/fuse solutions out there that, as far as I know, have not been tried in this context.